docs/hymns/resurrection/ekesmarout.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": {
+    "cop": "",
+    "eng": "Blessed are You indeed",
+    "ara": ""
+  },
+  "items": [
+    {
+      "user": "people",
+      "text": {
+        "cop": [
+          ""
+        ],
+        "eng": [
+          "Blessed are You indeed, with Your good Father, and the Holy Spirit, for You have risen and saved us. "
+        ],
+        "ara": [
+          ""
+        ]
+      }
+    }
+  ]
+}
```

Comment:
A lot of these files are missing the EOL character. Add this to your user settings in VSCode: `"files.insertFinalNewline": true`